packages/type-storage/src/wellKnownKeys.ts

```diff
@@ -0,0 +1,56 @@
+import { StorageFunctionModifier, StorageFunctionType } from '@polkadot/api-codec/Metadata';
+import Text from '@polkadot/api-codec/Text';
+import U8a from '@polkadot/api-codec/codec/U8a';
+import Vector from '@polkadot/api-codec/codec/Vector';
+
+import { createFunction } from './utils/createFunction';
+
+interface WKKMetadata {
+  documentation: Vector<Text>;
+  name: Text;
+  type: StorageFunctionType;
+}
+
+// Small helper function to factorize code on this page.
+const createWKKFunction = (prefix: Text | U8a, functionMetadata: WKKMetadata) =>
+  // TODO The expected 2nd argument is a StorageFunctionMetadata, but we actually only need the fields `documentation`
+  // and `type` of it, so in order to not input other fields (byteLength, fromJSON...) we lazily cast as any.
+  createFunction(prefix, { ...functionMetadata, modifier: new StorageFunctionModifier().fromJSON(0) } as any, {
```

Comment:
Would prefer some whitespace here, i.e. key per line, it is getting a bit unwieldly